components/supervisor/frontend/src/index.ts

```diff
@@ -30,6 +30,12 @@ window.addEventListener('error', (event) => {
             labels['error'] = 'LoadError';
         }
     }
+    if (event.error) {
```

Comment:
we should replace cases when error is missing as well, we can add info about source and rest of metadata with generic error messages like `LoadError` or `UnknownError`. It will help us to track them down.